boards/arm/pico_pi_m4/pinmux.c

```diff
@@ -0,0 +1,187 @@
+/*
+ * Copyright (c) 2018, Joris Offouga
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+
+#include <init.h>
+#include "device_imx.h"
+
+static int pico_pi_m4_pinmux_init(struct device *dev)
+{
+	ARG_UNUSED(dev);
+
+ #ifdef CONFIG_UART_IMX_UART_2
+	IOMUXC_SW_MUX_CTL_PAD_UART2_RX_DATA =
+		IOMUXC_SW_MUX_CTL_PAD_UART2_RX_DATA_MUX_MODE(0);
+	IOMUXC_SW_MUX_CTL_PAD_UART2_TX_DATA =
+		IOMUXC_SW_MUX_CTL_PAD_UART2_TX_DATA_MUX_MODE(0);
+	IOMUXC_SW_PAD_CTL_PAD_UART2_RX_DATA =
+		IOMUXC_SW_PAD_CTL_PAD_UART2_RX_DATA_PE_MASK  |
+		IOMUXC_SW_PAD_CTL_PAD_UART2_RX_DATA_PS(3)    |
+		IOMUXC_SW_PAD_CTL_PAD_UART2_RX_DATA_HYS_MASK |
+		IOMUXC_SW_PAD_CTL_PAD_UART2_RX_DATA_DSE(0);
+
+	IOMUXC_SW_PAD_CTL_PAD_UART2_TX_DATA =
+		IOMUXC_SW_PAD_CTL_PAD_UART2_TX_DATA_PE_MASK  |
+		IOMUXC_SW_PAD_CTL_PAD_UART2_TX_DATA_PS(3)    |
+		IOMUXC_SW_PAD_CTL_PAD_UART2_TX_DATA_HYS_MASK |
+		IOMUXC_SW_PAD_CTL_PAD_UART2_TX_DATA_DSE(0);
+
+	IOMUXC_UART2_RX_DATA_SELECT_INPUT =
+		IOMUXC_UART2_RX_DATA_SELECT_INPUT_DAISY(2);
+ #endif  /* CONFIG_UART_IMX_UART_2 */
+
+
+ #ifdef CONFIG_UART_IMX_UART_5
```

Comment:
This should be `CONFIG_I2C_4`

Reply:
Nevermind, you are right. It's confusing to see all the I2C macros below, but that's the name of the pad.